worldedit-core/src/main/java/com/fastasyncworldedit/core/queue/implementation/blocks/CharSetBlocks.java

```diff
@@ -288,4 +306,48 @@ public IChunkSet reset() {
         return null;
     }
 
+    @Override
+    public char[] load(final int layer) {
+        checkLayer(layer);
+        return super.load(layer);
+    }
+
+    @Override
+    public int getLayerCount() {
+        return layers;
+    }
+
+    private void checkLayer(int layer) {
```

Comment:
Method name does not really tell what's going on here

Reply:
Documentation needed.